server/project/parse.ts

```diff
@@ -77,3 +78,40 @@ export function parseRouterFunctions(code: string) {
 
   return routerFunctions;
 }
+
+export function createOperation(code: string) {
+  const routeRegExp =
+    /@Router\.(get|post)\("([^"]+)"\)\s+async\s+(\w+)\(((?:\s*\w+\s*:\s*\w+\s*,?\s*)+)\)\s*{/;
+  const match = routeRegExp.exec(code);
+
+  if (match) {
+    const method = match[1];
+    const endpoint = match[2];
+    const functionName = match[3];
+    const params = match[4]
+      .split(",")
+      .map((param) => {
+        const paramName = param.trim().split(":")[0];
+        return paramName;
+      })
+      .filter((param) => param !== "" && !param.startsWith("session"));
```

Comment:
Why `session` is a special case?

Reply:
I'm using these params to generate the input fields the user sees on the testing client frontend. I remove session params because the user shouldn't input a session.